hallux/backends/litellm.py

```diff
@@ -38,10 +38,7 @@ def query(self, request: str, issue: IssueDescriptor | None = None, issue_lines:
         if not self.valid:
             return []
 
-        logger.debug("")
-        logger.debug("[LiteLLM REQUEST]:")
-        for line in request.split("\n"):
-            logger.debug(line)
+        logger.log_multiline("[LiteLLM REQUEST]:", request, "debug")
```

Comment:
it used to be DEBUG-only log, but now it looks like it will log all the time. Can you make it debug-only again?